components/datasource-selector/partials/datasource_selector.html

```diff
@@ -1,4 +1,4 @@
-<div class="card panel-default hs-main-panel">
+<div class="card hs-main-panel">
```

Comment:
Ah, you want some merge conflicts with #957 ? 😆

Reply:
Don't think I can do much a bout it now. Could create another commit which puts back the class, but the merge conflict would still happen at some moment when rebasing.

Reply:
It's fine. I will resolve the merge conflict when rebasing.